ReactNativeClient/lib/time-utils.js

```diff
@@ -111,6 +111,18 @@ class Time {
 	sleep(seconds) {
 		return this.msleep(seconds * 1000);
 	}
+
+
+	goBackInTime(n, timeDuration) {
+		// Note that we are starting from the first ms of the current timeDuration
+		// eg. If we go back by one day we are subtracting (24*60*60*1000) ms from the start ms of today
+		return moment().startOf(timeDuration).subtract(n, timeDuration).format('x');
+	}
+
+	goForwardInTime(n, timeDuration) {
+		return moment().startOf(timeDuration).add(n, timeDuration).format('x');
+	}
```

Comment:
As a first argument to these function, please pass the date that should go forward/backward. Also please clarify what is "n" (possible values, unit) and what is timeDuration (possible values, unit, as from your code it seems to be "day", "hours", etc. but from your example it seems to be milliseconds).

Reply:
I didn't find tests for these two functions. Please add some if you haven't already done so.

Reply:
OK. I modified the function and added clarification.

Where should I place the tests? I can't find any existing tests for timeUtils functions.

Also, do we need a test here as we're just calling moment.js functions as per its doc? I haven't written any additional logic.

Reply:
In that case please create a new file for it. For example duplicate tests/ArrayUtils.js and rename to tests/timeUtils.js. Even though it's relatively straightforward implementation of the moment.js api, it's still good to have a test units to specify what cases exactly we support. It would also be useful if for whatever reason we change from moment.js to another time library.